scripts/PHNI-JP/c101203022.lua

```diff
@@ -0,0 +1,77 @@
+--アロマリリスーロザリーナ
+local s,id,o=GetID()
+function c101203022.initial_effect(c)
+	--recover 
+	local e1=Effect.CreateEffect(c)
+	e1:SetDescription(aux.Stringid(id,0))
+	e1:SetType(EFFECT_TYPE_QUICK_O)
+	e1:SetCode(EVENT_FREE_CHAIN)
+	e1:SetRange(LOCATION_HAND)
+	e1:SetCountLimit(1,id)
+	e1:SetCost(s.cost)
+	e1:SetTarget(s.retg)
+	e1:SetOperation(s.recop)
+	c:RegisterEffect(e1)
+	--spsummon
+	local e2=Effect.CreateEffect(c)
+	e2:SetDescription(aux.Stringid(id,0))
```

Comment:
Duplicate string reference
```suggestion
	e2:SetDescription(aux.Stringid(id,1))
```